models/metrics/for_anomalies/re_data_last_stats.sql

```diff
@@ -6,7 +6,7 @@ with median_value as (
         column_name,
         metric,
         interval_length_sec,
-        avg(value) {% if target.type != 'postgres' %} over(partition by {{ columns_to_group_by }}) {% endif %} as last_avg,
```

Comment:
To not repeat and make it more extensible, I think macro with those 2 db would be nice. So that we could write something like this:

```
{% if target.type in postgres_type_db() %}
```

Reply:
Agree, sounds reasonable. Do you have any recommendations where to place this macro?
For me `macros/db/postgres` is the best folder for it.

Reply:
done